<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indicateurs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>11</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Domaine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>Composition Parc</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Delais</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Temperature</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Volumes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cartographie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Afficheurs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Masses/Temps</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Date debut</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Date de fin</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCalendarWidget" name="dateEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCalendarWidget" name="dateEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Items</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Resultats</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport_parc">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_parc"/>
    <addaction name="actionExport_indicateurs"/>
   </widget>
   <addaction name="menuExport_parc"/>
  </widget>
  <action name="actionExport_parc">
   <property name="text">
    <string>Export_parc</string>
   </property>
  </action>
  <action name="actionExport_indicateurs">
   <property name="text">
    <string>Export_indicateurs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
